### PR TITLE
00015574_unable_retrieve_final_1

### PR DESCRIPTION
**Pull request by VitalySerebrekov**
00015574_final_1

Link to the case this pull request closes
------------------------------------------
[Case number](https://link-to-case.here)

What does this implement/fix? Explain your changes.
---------------------------------------------------
**What is the current behavior?**

**What is the new behavior?**

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- [ ] Yes
- [ ] No

Any relevant logs, error output, etc?
-------------------------------------
…

Any other comments?
-------------------
…

### CODE REVIEW

File: src/routes/unlocked-packages/create-snapshot/helper.js [hunk 1]
```diff
@@ -10,9 +10,17 @@ const { MetadataTypeParser } = require('./metadataTypeParser');
 function retrievePackages(accessToken, projectName, packageName, dependencyList, log) {
   return new Promise((resolve, reject) => {
     try {
+
+      //  Case 00015574
+      const packageNameFolder = packageName.replaceAll(/\//g, "-");
```
Comment (Dm1tryKarpenko) [inline src/routes/unlocked-packages/create-snapshot/helper.js]: давай одинарные ковычки, везде по проекту одинарные

File: src/routes/unlocked-packages/create-snapshot/metadataTypeParser.js [hunk 2]
```diff
@@ -346,16 +348,20 @@ class MetadataTypeParser {
...
         this.zip.addFile(`${folderType}/CustomLabels.labels`, full);
         this.updateChunkList('CustomLabel');
         fullLabelXML = '';
       }
     });
 
     if (this.componentList.length) {
-      const full = `${header}${fullLabelXML}${footer}`;
+      // const full = `${header}${fullLabelXML}${footer}`;
+      const full = xmlFormat(`${header}${fullLabelXML}${footer}`, {collapseContent: true, lineSeparator: '\n'});
```
Comment (Dm1tryKarpenko) [inline src/routes/unlocked-packages/create-snapshot/metadataTypeParser.js]: { collapseContent: true, lineSeparator: '\n' }
пробелы возле ковычек

File: src/routes/unlocked-packages/create-snapshot/helper.js (hunk 1 above)
Comment (Dm1tryKarpenko) [inline src/routes/unlocked-packages/create-snapshot/helper.js]: давай это в контроллер вынесем

**Review by Dm1tryKarpenko — CHANGES_REQUESTED** [review]

File: src/routes/unlocked-packages/create-snapshot/helper.js (hunk 1 above)
Comment (VitalySerebrekov) [reply src/routes/unlocked-packages/create-snapshot/helper.js]: resolved

**Review by Dm1tryKarpenko — APPROVED** [review]

**Pull request merged**